Tests/Microsoft.Teams.Api.Tests/Activities/Invokes/MessageExtensions/CardButtonClickedMEActivityTests.cs

```diff
@@ -0,0 +1,140 @@
+            Name = "Somelist",
+            Value = "Toronto"
+        };
+        return new CardButtonClickedActivity()
+        {          
+            Value = anyValueObject,
+            Conversation = new Conversation()
+            {
+                Id = "19:someid",
+                Type = ConversationType.Personal 
+            },
+            ServiceUrl = "https://url-value"
+        };
+    }
+
+    [Fact]
+    public void CardButtonClickedMEActivity_JsonSerialize()
+    {
+        var activity = setupCardButtonClickedActivity();
+
+        var json = JsonSerializer.Serialize(activity, new JsonSerializerOptions()
+        {
+            WriteIndented = true,
+            IndentSize = 2,
+            DefaultIgnoreCondition = System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull
+        });
+
+        string expectedPath = "Activity.Invoke.ComposeExtension/onCardButtonClicked";
+        Assert.Equal(expectedPath, activity.GetPath());
+        Assert.Equal(File.ReadAllText(
+            @"../../../Json/Activity/Invokes/CardButtonClickedMEActivity.json"
+        ), json);
+    }
+
+    [Fact]
+    public void CardButtonClickedMEActivity_JsonSerialize_Derived()
+    {
+        MessageExtensionActivity activity = setupCardButtonClickedActivity();
+
+        var json = JsonSerializer.Serialize(activity, new JsonSerializerOptions()
+        {
+            WriteIndented = true,
+            IndentSize = 2,
+            DefaultIgnoreCondition = System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull
+        });
+
+        string expectedPath = "Activity.Invoke.ComposeExtension/onCardButtonClicked";
+        Assert.Equal(expectedPath, activity.GetPath());
+        Assert.Equal(File.ReadAllText(
+            @"../../../Json/Activity/Invokes/CardButtonClickedMEActivity.json"
+        ), json);
+    }
+
+    [Fact]
+    public void CardButtonClickedMEActivity_JsonSerialize_Derived_Interface()
+    {
+        InvokeActivity activity = setupCardButtonClickedActivity();
+
+        var json = JsonSerializer.Serialize(activity, new JsonSerializerOptions()
+        {
+            WriteIndented = true,
+            IndentSize = 2,
+            DefaultIgnoreCondition = System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull
+        });
+
+        string expectedPath = "Activity.Invoke.ComposeExtension/onCardButtonClicked";
+        Assert.Equal(expectedPath, activity.GetPath());
+        Assert.Equal(File.ReadAllText(
+            @"../../../Json/Activity/Invokes/CardButtonClickedMEActivity.json"
+        ), json);
+    }
+
+    [Fact]
+    public void CardButtonClickedMEActivity_JsonDeserialize()
+    {
+        var json = File.ReadAllText(@"../../../Json/Activity/Invokes/CardButtonClickedMEActivity.json");
+        var activity = JsonSerializer.Deserialize<CardButtonClickedActivity>(json);
+        var expected = setupCardButtonClickedActivity();
+
+        Assert.Equal(expected.ToString(), activity.ToString());
```

Comment:
can you make sure to resolve any of these `Dereference of a possibly null reference.` warnings and also run `dotnet format` please?